pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cpython-lldb"
-version = "0.2.0"
+version = "0.0.0"
```

Comment:
Wow.. how does it work?!

Reply:
You won't like the answer! :rofl: Basically, I ask poetry to update this file right before I create a release: https://github.com/malor/cpython-lldb/blob/master/.github/workflows/release.yml#L42 . So the wheel will have the correct version, but if you just try to install the package by cloning this repo and running `poetry install`, the version number will be 0.0.0

Reply:
Ah.. I see.. so this is your way to use Git version during publishing and prohibit installing the package from sources directly. Is that right?

Reply:
Yeah, sort of. I don't like it, but this appears to be the most straightforward way to manage versions via git tags.